### DISCUSSION TITLE
Trouble connecting with EMQX.io tool

### DISCUSSION THREAD

**Discussion by dkanday** [emqx/emqx#5475] [Q&A]
Hi there,

I am new to EMQX and I am trying to use the MQTT broker to connect to my Arduino. Basically I am trying to send info from a website to the EMQX and then this info would be relayed over to the Arduino. This information would be to change the speed of the motor based on the user input.
When I am using the tool: http://tools.emqx.io/recent_connections/5181026a-52e6-47f1-b925-86655a923451 and try to create a new connection. I add in all the parameters and try to publish or subscribe, it says connection failed. Is there something basic that I could be missing?

**Comment by qzhuyan** [comment]
|  I add in all the parameters and try to publish or subscribe, it says connection failed.
Could you provide the parameters in use?
